assemblies/upgrading-odh-v1.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+:context: upgradev1
+
+[id="upgrading-odh-v1_{context}"]
+= Upgrading Open Data Hub version 1
+
```

Comment:
should this be 
[id="upgrading-odh-v1-to-v2_{context}"]
= Upgrading Open Data Hub version 1 to version 2

Reply:
to clarify it is not v1 to v1
future assemblies could be v2 to v2